pkg/views/util/table.go

```diff
@@ -3,15 +3,71 @@
 
 package util
 
-import "regexp"
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"regexp"
+
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/daytonaio/daytona/pkg/views"
+	"golang.org/x/term"
+)
 
 var AdditionalPropertyPadding = "  "
 
 // Left border, BaseTableStyle padding left, additional padding for workspace name and target, BaseTableStyle padding right, BaseCellStyle padding right, right border
 var RowWhiteSpace = 1 + 4 + len(AdditionalPropertyPadding)*2 + 4 + 4 + 1
 var ArbitrarySpace = 10
 
-func GetTableMinimumWidth(data [][]string) int {
+func GetTableView(data [][]string, headers []string, footer *string, fn interface{}, args ...interface{}) string {
+	re := lipgloss.NewRenderer(os.Stdout)
+
+	terminalWidth, _, err := term.GetSize(int(os.Stdout.Fd()))
+	if err != nil {
+		fmt.Println(data)
+		return ""
+	}
+
+	breakpointWidth := views.GetContainerBreakpointWidth(terminalWidth)
+
+	minWidth := getMinimumWidth(data)
+
+	if breakpointWidth == 0 || minWidth > breakpointWidth {
+		// Fallback to unstyled view
+		function := reflect.ValueOf(fn)
```

Comment:
Seems a bit over-engineered 😅 

Can't we have just `fallbackRender func()` and then pass
```golang
func() {
  renderUnstyledList(list, ...)
}
```
from the calling function.

Also, add a comment that explains when the fallback render is used.

Reply:
I see, yes we can